ci/docker/cuda11.0/Dockerfile-x86_64

```diff
@@ -75,7 +75,7 @@ ENV LIBRARY_PATH /usr/local/cuda/lib64/stubs
 # assume we are building container in the root of the git repo...
 COPY . /io
 WORKDIR /io
-RUN make
+RUN make target=manylinux2010
```

Comment:
Did you just mean `manylinux` here?

Reply:
Right. This is why the 2014 wheels ended up with AVX2…